.storybook/stories/signpost/signpost.stories.ts

```diff
@@ -53,6 +53,7 @@ const template = `
   <div style="padding: 250px; text-align: center">
     <clr-signpost>
       <clr-signpost-content [clrPosition]="clrPosition">
+        <clr-signpost-title>Title</clr-signpost-title>
```

Comment:
Add a binding so it can be changed trough storybook controls
```suggestion
        <clr-signpost-title>{{title}}</clr-signpost-title>
```